jestEnvVar.js

```diff
@@ -0,0 +1,2 @@
+process.env.PGDATABASE= 'nc_news_test'
+process.env.PGPASSWORD= 'Goku@5050'
```

Comment:
Bear in mind that this file is not gitignored!
Out of interest, why have you needed to introduce this?
You should find that this logic is already handled in `db/conection.js`

Reply:
Jest couldn't find any environment variables so I created this file to use--setupFiles flag. Couldn't find any other way to do that.